build_windows.py

```diff
@@ -145,45 +148,46 @@
 		)
 	# Copy some theme icons
 	sizes = ["16x16", "24x24", "32x32", "scalable"]
-	icons = {	"status"  : [
-					"image-missing",
```

Comment:
The reason these look all weird and inconsistent is because it's mixing spaces and tabs.

I assume your preference is tabs? Technically speaking, pretty much every Python style guide recommends using spaces exclusively, because the interpreter can sometimes get confused about indentation levels when the two get mixed, and also to prevent stuff like ^.

(But I'm not going to press this or anything, it's your project)

Reply:
Yes, but mixing them is something I'm trying to not do. But build_windows.py script is little special, I've edited it few times with editors that are not good at keeping indentation :)